create_bags/helpers.py

```diff
@@ -0,0 +1,82 @@
+from os import listdir
+from os.path import isfile, join
+from shutil import copy2
+
+
+def format_aspace_date(dates):
```

Comment:
For future you, maybe a bit more detail on what's happening here (i.e. inferring month and day)